client/state/data-layer/wpcom/sites/comments/index.js

```diff
@@ -8,13 +8,48 @@ import { forEach, groupBy } from 'lodash';
  * Internal dependencies
  */
 import { mergeHandlers } from 'state/action-watchers/utils';
-import { COMMENTS_LIST_REQUEST, COMMENTS_RECEIVE } from 'state/action-types';
+import { COMMENTS_LIST_REQUEST, COMMENTS_RECEIVE, COMMENT_REQUEST } from 'state/action-types';
 import { http } from 'state/data-layer/wpcom-http/actions';
 import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 import replies from './replies';
 import likes from './likes';
 import { errorNotice } from 'state/notices/actions';
 import { getRawSite } from 'state/sites/selectors';
+import { getSiteName as getReaderSiteName } from 'reader/get-helpers';
+import { getSite as getReaderSite } from 'state/reader/sites/selectors';
+
+export const requestComment = ( store, action ) => {
+	const { siteId, commentId } = action;
+	store.dispatch(
+		http( {
+			method: 'GET',
+			path: `/sites/${ siteId }/comments/${ commentId }`,
+			apiVersion: '1.1',
+		} ),
+	);
+};
+
+export const receiveCommentSuccess = ( store, action, next, response ) => {
+	const { siteId } = action;
+	const postId = response && response.post && response.post.ID;
+	store.dispatch( {
+		type: COMMENTS_RECEIVE,
+		siteId,
+		postId,
+		comments: [ response ],
+	} );
+};
+
+export const receiveCommentError = store => {
+	const site = getReaderSite( store.getState() );
+	const siteName = getReaderSiteName( site );
+
+	store.dispatch(
+		translate( 'Failed to retrieve comment for site “%(siteName)s”', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Failed to retrieve comments for site “%(siteName)s”' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/comments/get-by-commentid)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).